### PR TITLE
Fix workshop

### PR DESCRIPTION
**Pull request by jc-cb**

### CODE REVIEW

**Review by rcbgr — COMMENTED** [review]
We only need one THIRD-PARTY-NOICES LICENSE file per rep. Please remove .DS_Store

File: FIX/environment.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+name: PrimeFIX
```
Comment (rcbgr) [inline FIX/environment.yml:1]: We don't need this if we have a requirements.txt, correct?

**Comment by nmorgancb** [comment]
Not a blocker: can you lowercase the main folder to `fix` and rename the subfolder? It feels weird to have FIX/fix

**Comment by jc-cb** [comment]
made these changes, please +2

**Review by nmorgancb — APPROVED** [review]

**Review by rcbgr — COMMENTED** [review]
LGTM

**Pull request merged**